.github/workflows/build-and-test.yml

```diff
@@ -12,14 +12,14 @@ jobs:
     name: Run setup.py phases needed by spack
     needs: [setup_test]
     runs-on: ubuntu-latest
-    container: precice/precice
```

Comment:
I'm not a big fan of this change. Why is it necessary? Be aware that you will need to update this for every single release of preCICE. Additionally: You won't be testing against the develop version of preCICE, but against the last released version. This leaves you with a blind spot between the latest release and the develop version and might lead to bad surprises when you update.

However, I guess you cannot do a lot here. Ideally there would be a `precice/precice:develop` pointing to the develop version of preCICE and a `precice/precice:latest` pointing to the latest release. Not sure which one you would need here.

Reply:
Good catch! This was leftover from when I was testing things on my local machine using `act`. Indeed one does not need a version tag here!